dpdd.tex

```diff
@@ -1068,6 +1091,82 @@ \subsubsection{\ForcedSource Table}
+
+shearObjectId & uint64 & ~ & Unique object identifier. \\
+
+cellId & uint64 & ~ & Unique identifier for the coadd cell in which this object was detected and measured. \\
+
+metaStep & int8 & ~ & Enumeration value for which of the five \texttt{Metacalibration} modifications was applied to images prior to all measurements. \\
+
+radec & double[2] & degrees & Position of the object (centroid), measured jointly across all bands. \\
+
+maskFractionObj & float & ~ & Weighted fraction of this object's pixels that were masked, over all visit images input to the coadd. \\
+
+maskFractionCell & float & ~ & Weighted fraction the coadd cell's that were masked, over all visit images input to the coadd.\footnote{This and \texttt{nEpochCell} are the same for all rows with the same \texttt{cellId}, but are expected to be access per-object; they may or may not be normalized into a separate table in the physical schema.}. \\
+
+nEpochCell & int32[ugrizy] & ~ & Number of input visits that contributed to this coadd cell. \\
+
+g1 & float & ~ & Ellipticity from Gaussian-weighted moments, on-axis, measured consistently over central\footnote{Probably $riz$} bands. \\
+
+g2 & float & ~ & Ellipticity from Gaussian-weighted moments, off-axis. \\
+
+gCov & float[3] & ~ & Covariance matrix for (\texttt{g1}, \texttt{g2}). \\
+
+T & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) \\
+
+SNR & float & ~ & Gaussian weighted signal-to-noise ratio. \\
+
+TErr & float & arcsec$^{2}$ & Uncertainty on \texttt{T}.\\
+
+g1PSFMeta & float & ~ & PSF model ellipticity from Gaussian-weighted moments, on-axis, computed from the central-band-averaged metacalibration-modified PSF models.\footnote{This and other PSF quantities are not constant over a cell only because the PSF may depend on the SED of the object.} \\
+
+g2PSFMeta & float & ~ & PSF model ellipticity from Gaussian-weighted moments, off-axis, computed from the central-band-averaged metacalibration-modified PSF models. \\
+
+TPSFMeta & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) of the central-band-averaged metacalibration-modified PSF models. \\
+
+g1PSFOrig & float & ~ & PSF model ellipticity from Gaussian-weighted moments, on-axis, computed from the central-band-averaged original PSF models. \\
+
+g2PSFOrig & float & ~ & PSF model ellipticity from Gaussian-weighted moments, off-axis, computed from the central-band-averaged original PSF models. \\
+
+TPSFOrig & float & arcsec$^{2}$ & Gaussian-weighted moments trace (Ixx + Ixy) of the central-band-averaged original PSF models. \\
+
+stdFlux & float[ugrizy] & nJy & PSF-corrected aperture fluxes, measured using an approach similar to the ``standard color'' in the main \Object table.\footnote{This is not expected to be a good estimate of total flux, but \emph{some} estimate of total flux is useful, so this is still reported as fluxes, as opposed to the colors in \Object's \texttt{stdColor}.} \\
+
```

Comment:
I'd be a bit careful on the "good measure of total flux" here. We have to use this flux for various bits of photo-z work to account for selections.

Reply:
If you're trying to measure the total flux of a galaxy of unknown size, I think you can't have a fixed aperture, and my understanding is that this does.  It may well be pretty decent as a total flux for the small, faint galaxies that form the source sample in WL.

Reply:
Oh sorry. I'm not arguing the physical point. I am only saying that we might do the less ideal thing.